cumulusci/core/tasks.py

```diff
@@ -88,6 +90,16 @@ def _init_options(self, kwargs):
         if kwargs:
             self.options.update(kwargs)
 
+        # Handle default values
+        for name, config in list(self.task_options.items()):
+            if name not in self.options and "default" in config:
+                self.options[name] = config["default"]
```

Comment:
Should probably do a deepcopy here. In case there are mutable defaults and multiple instances of the same task class in a flow.